pajbot/apiwrappers/twitch/tmi.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from pajbot.apiwrappers.twitch.base import BaseTwitchAPI
+
+log = logging.getLogger(__name__)
+
+
+class TwitchTMIAPI(BaseTwitchAPI):
+    authorization_header_prefix = "OAuth"
```

Comment:
Don't think you need Client-ID, or `Authorization` headers with the `tmi.twitch.tv` API. You could just extend from BaseAPI, and save some overhead with the client credentials/authorization stuff.